pkg/kv/kvserver/kvstorage/testdata/init

```diff
@@ -0,0 +1,39 @@
+# Uninitialized deprecated replica: doesn't have a RaftReplicaID, it's just a
+# HardState. We expect this to be removed by load-and-reconcile.
+new-replica range-id=5
```

Comment:
r5 doesn't show up in any way in the output below, except in comments that reason about what happened to it. Is there a way to surface it in the output? Or is the absence of r5 in the output the intended signal?

Reply:
It does show up:

<img width="652" alt="image" src="https://user-images.githubusercontent.com/5076964/216624048-eb226e16-c994-489b-a201-a9e88f26d761.png">

Basically it was tossed out.